web-services-metadata/2.1/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Jakarta Web Services Metadata Specification 2.1"
+date: 2019-07-23T14:28:20+0200
+summary: "Initial release of Jakarta Web Services Metadata"
+---
+Jakarta Web Services Metadata defines a programming model for Web Services in Java,
+use of metadata, a non-normative processing model for metadata annotated web service
+source files, runtime requirements for a container, and annotations used for WSDL,
+binding, and configuration.
+
+* [Jakarta Web Services Metadata 2.1 Specification Document](./jakarta.jws-spec-2.1.pdf) (PDF)
```

Comment:
Sorry, I missed this before, but the name of the spec file should not have the "jarkata." in the name.  According to [naming doc](https://github.com/jakartaee/specification-committee/blob/master/names.adoc), it should be ws-metadata-spec-2.1.pdf.

Reply:
done